userguide/jails.rst

```diff
@@ -663,14 +663,22 @@ example is shown in :numref:`Figure %s <jail_overview_fig>`.
    +-------------------+----------------------------------------------------+
 
 
+Select multiple jails from the list by setting the leftmost checkboxes.
+Options appear to |ui-jail-start| or |ui-jail-stop|, |ui-jail-update|,
```

Comment:
"Options appear to" is kind of a strange wording.  "appear to" can be misinterpreted as "it looks like they do this".

So maybe:
"After selecting one or more jails, icons appear which can be used to start, stop, update, or delete those jails."